### ISSUE TITLE
Documentation for dbt-spark

### ISSUE THREAD

**Issue by ajithshetty** [astronomer/astronomer-cosmos#340]
The documentation is not fully clear when it comes to using the dbt spark using thrift.
https://astronomer.github.io/astronomer-cosmos/dbt/install-options.html

Looking at the code and a [profile page](https://astronomer.github.io/astronomer-cosmos/dbt/connections-profiles.html) made sure that the cosmos does support Spark Thrift.
But the documentations lacks the information.
Can we have please more detailed steps.

**Comment by jlaneve** [comment]
Hi @ajithshetty, thanks for opening this issue! I'm curious what you'd like to see here - a step-by-step guide on how to set up Spark itself, or just a guide on how to configure the connection for use in Cosmos?

**Comment by ajithshetty** [comment]
Thank you @jlaneve for your response.
While I was trying to get the spark thrift working using the local mode, I couldn't find the enough documents to support it.
There were few libraries I had to install specifically while using MWAA using startup.sh. not sure if it is specific to MWAA or not.

A step by step guide to setup the connection would definitely save a day of search effort.

**Comment by jensenity** [comment]
I see. I don't think this is just a dbt-spark configuration guide documentation request. I think in general right now astronomer cosmos docs doesn't have a friendlier step by step guide, when I first used cosmos, I had to study the whole code base in order to use it for my use case. And when I added the spark thrift connection profile, I just followed the other connection profiles. We can work together to build a step by step guide together @ajithshetty, since im the one that contributed on the spark profile. Lets talk on the slack channel?
